Benchmarks/Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version: 5.10
```

Comment:
Just for consistency since we've decided to set the tools and package version to 5.8 I'd like to set this here as well

Reply:
All set! Included in a follow-on commit